sos/plugins/unpackaged.py

```diff
@@ -0,0 +1,86 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin
+import os
+import re
+
+
+class Unpackaged(Plugin, RedHatPlugin):
+    '''
+    Collects a list of files that are not handled by the package
+    manager
+    '''
+
+    def setup(self):
+
+        def all_files_rpm():
+            """
+            It returns a list of files known by the package manager
+            """
+            return self.get_command_output('rpm -qal')['output'].splitlines()
```

Comment:
It would be preferable to get this from the existing `Packages` dictionary in `Plugin->policy()->PackageManager`. This is already in a distribution agnostic format and it avoids re-running the `rpm` program (and all the side effects that causes).

It would also be a step toward making this distribution neutral.